src/model/CoilSystemCoolingWater.hpp

```diff
@@ -65,15 +65,16 @@ namespace model {
 
     double minimumWaterLoopTemperatureForHeatRecovery() const;
 
-    boost::optional<WaterToAirComponent> companionCoilUsedForHeatRecovery() const;
+    boost::optional<HVACComponent> companionCoilUsedForHeatRecovery() const;
```

Comment:
Technically the return type could stay WaterToAirComponent because the only supported type "AT THE MOMENT" is CoilCoolingWater. And I don't want to have to break API later on, so I also changed it.